examples/03_connectivity/plot_compare_decomposition.py

```diff
@@ -1,26 +1,32 @@
 """
-Dictionary Learning and ICA for doing group analysis of fMRI
-============================================================
+Deriving spatial maps from group fMRI data using ICA and Dictionary Learning```
+===============================================================================
 
-This example applies dictionary learning and ICA to
-movie-watching fMRI data,
-visualizing resulting components using atlas plotting tools.
+Various approaches exist to derive spatial maps or networks from 
+group fmr data. The methods extract distributed brain regions that 
+exhibit similar BOLD fluctuations over time. Decomposition
+methods allow for generation of many independent maps simultaneously 
+without the need to provide a priori information (e.g. seeds or priors.) 
 
-Dictionary learning is a sparsity based decomposition method for extracting
-spatial maps. It extracts maps that are naturally sparse and usually cleaner
-than ICA
+This example will apply two popular decomposition methods, ICA and 
+Dictionary Learning, to fMRI data measured while children and young adults 
+watch movies. The resulting maps will be visualized using atlas plotting 
+tools. 
 
-   * Arthur Mensch et al. `Compressed online dictionary learning for fast resting-state fMRI decomposition
-     <https://hal.archives-ouvertes.fr/hal-01271033/>`_,
-     ISBI 2016, Lecture Notes in Computer Science
+CanICA is an ICA method for group-level analysis of fMRI data. Compared
+to other strategies, it brings a well-controlled group model, as well as a
+thresholding algorithm controlling for specificity and sensitivity with
+an explicit model of the signal. The reference papers are:
 
-.. note::
+    * G. Varoquaux et al. "A group model for stable multi-subject ICA on
+      fMRI datasets", NeuroImage Vol 51 (2010), p. 288-299
+
+    * G. Varoquaux et al. "ICA-based sparse features recovery from fMRI
+      datasets", IEEE ISBI 2010, p. 1177
```

Comment:
I would rather remove that citation, which isn't that interesting, and cite the Mensch paper, that describes the strategy actually used for dictionary learning.


"""
   * Arthur Mensch et al. `Compressed online dictionary learning for fast resting-state fMRI decomposition
     <https://hal.archives-ouvertes.fr/hal-01271033/>`_,
     ISBI 2016, Lecture Notes in Computer Science
"""

Reply:
Actually: brain fart: remove the second citation, but don't add the Mensch citation, it goes below, in the dictionary learning part.

Reply:
Right so that citation is referenced later on when we bring up Dictionary Learning. Would you rather I move it back up top, keeping all the references in the same place?

Reply:
No, I just made a mistake. Sorry.